### PR TITLE
[FIX] Resend email without unlock (RT-3104)

### PR DESCRIPTION
**Pull request by darkdarkdragon**
Sign-up: resend email without requiring unlock

### CODE REVIEW

File: src/js/tabs/register.js [hunk 1]
```diff
@@ -137,39 +137,46 @@ RegisterTab.prototype.angular = function (module) {
       });
     };
 
+    var _resnedEmail = function(masterkey) {
```
Comment (rht) [inline src/js/tabs/register.js]: typo?

File: src/js/tabs/register.js [hunk 2]
```diff
@@ -137,39 +137,46 @@ RegisterTab.prototype.angular = function (module) {
...
-          id.resendEmail({
-            id:$scope.userBlob.id,
-            url:$scope.userBlob.url,
-            username: $scope.userCredentials.username,
-            account_id: $scope.userBlob.data.account_id,
-            email: $scope.newEmail || $scope.userBlob.data.email,
-            masterkey: masterkey
-          }, function(err, response){
+      if ($scope.keyOpen) {
+        _resnedEmail($scope.keyOpen);
```
Comment (rht) [inline src/js/tabs/register.js]: If typo "happens" 3 times in a row, either you mass-rename the parameter, or you used auto-complete to type this.

**Comment by mrajvanshy** [comment]
LGTM

**Comment by yongsoo** [comment]
LGTM

**Pull request merged**